cli/commands/patchMigrations.js

```diff
@@ -0,0 +1,73 @@
+var Promise = require('bluebird');
+
+var fs = Promise.promisifyAll(require('fs'));
+
+let frameworkName = 'Transmute Framework'
+
+let patchBegin = `🦄 ${frameworkName}`
+let patchEnd = `🐩 ${frameworkName}`
+
+function replaceAll(str, find, replace) {
+  return str.replace(new RegExp(find, 'g'), replace);
+}
+
+const backupPatchTarget = (targetPath) =>{
+    let contents
+    return fs.readFileAsync(targetPath, "utf8")
+        .then((_contents) => {
+            contents = _contents
+            return fs.writeFileAsync(targetPath + '.transmute.bak', contents, {})
+        })
+        .then(() =>{
+            return contents
+        })
+}
+
+const buildPatch = (fileString) =>{
+    // Fix path to match result of copyTransmuteContracts
+    fileString = fileString.replace(/\.\//g, "./TransmuteFramework/");
```

Comment:
This is terrible i agree. but so is patching and unpatching instead of using a package manager. if only there were a package manager, i would call it ethpm.